resources/views/auth/info.blade.php

```diff
@@ -9,17 +9,18 @@
                     <div class="card-body">
                         <form method="post" action="">
                             @csrf
-                            <div class="form-group">
-                                <label class="col-sm-4 control" class="col-sm-4 control" for="name">@lang('user.name')</label>
-                                <div class="col-sm-8">
-                                <input class="form-control" type="text" name="name" id="name" required>
+                            <div class="form-row">
+                                <div class="form-group col-md-4">
+                                    <label class="col-sm-4 control" for="first_name">@lang('user.name')</label>
```

Comment:
The `@lang(user.name)` is not the one that you want to use.
You need first name here.

Reply:
fixed